### ISSUE TITLE
[IOS] Missing 'Cancel' and 'Continue' buttons  when login by FB app

### ISSUE THREAD

**Issue by hienhodeveloper** [thebergamo/react-native-fbsdk-next#533]
## 🐛 Bug Report 
When I log in with the Facebook app for the first time, 'Cancel' and 'Continue' buttons are missing, UI like broken, and I can't scroll either.
This bug also happen when request Connect to FB Pages 

Permissions Requests (FB allowed these):
["pages_messaging",
    "pages_manage_metadata",
    "pages_read_engagement",
    "pages_show_list",
    "pages_manage_engagement",
    "pages_read_user_content",
    "pages_manage_posts",
    "public_profile",
    "email"]

![image](https://github.com/thebergamo/react-native-fbsdk-next/assets/51476472/72e19243-bf3e-4c69-87ce-ceda77b37f36)

## To Reproduce
Run this code with these permissions
```
LoginManager.logInWithPermissions(
      permissions || [
        'pages_messaging',
        'pages_manage_metadata',
        'pages_read_engagement',
        'pages_show_list',
        'pages_manage_engagement',
        'pages_read_user_content',
        'public_profile',
        'email',
      ],
    )
```

## Expected Behavior
Show Cancel' and 'Continue' buttons, UI not broken

## Environment
IOS only
"react-native": "0.71.14",
"react-native-fbsdk-next": "^13.0.0"

**Comment by AndersonGuerraSantos** [comment]
This happens to me when I am not logged into Facebook on Safari.

**Comment by sufian1999** [comment]
Yes. This also happens with me when I try to login or connect with FB pages. I also tried to downgrade the sdk version for ios but couldn't do so.

**Comment by manhdev0901** [comment]
This only happens when logging in through the Facebook app. 
I have a temporary solution for iOS: You need to log in to Facebook via Safari first, and then log in again. This time, the login on Safari will show two options: 'Cancel' and 'Continue'.
If there are any other solutions, please leave a comment below and I will read it.

**Comment by sufian1999** [comment]
@manhdev0901 
Well, there is one more but that's not a good one. 
The user must delete the FB app and then the user can enter credentials in the modal provided by the FB SDK. There buttons won't hide.

I tried an app called Privyr, they also implemented the FB integration. There, FB SDK UI is of the previous version. But I couldn't find a way to downgrade the FB SDK for ios. I tried to set a version for FB SDK in the podfile but it gives errors when I do 'pod install'.
